start/scaling-up/scaling-up.md

```diff
@@ -0,0 +1,70 @@
+[title]: - "Scale Up your Workflow on OSG Connect"
+[TOC]
+
+## Background
+
+Much of OGS's computing power comes from the ability to run a large number 
+of jobs simulateously. Many HTC-friendly workflows involve running the same 
+job multiple times with no or minor changes between them. Here, we will 
+talk about how to submit multiple jobs at once using the `queue` command as 
+well as some important considerations and tips for scaling up your HTCondor jobs. 
+For a hands-on example of these concepts, checkout the [OSG Connect Quickstart]
+(https://support.opensciencegrid.org/support/solutions/articles/5000633410-osg-connect-quickstart). 
+
+## Things to Consider

```

Comment:
Maybe a different title? I think of "Things to Consider" as more about the actual jobs, not the terminology.

Reply:
oh, never mind, I wasn't looking at the header level carefully.